### ISSUE TITLE
Show layout on year view

### ISSUE THREAD

**Issue by wjdp** [newtheatre/history-project#135]
Single line list works but when lots of shows are in a year it quickly becomes hard to parse. Experiment with other view layouts:

## Grid layout (from [Plex](https://plex.tv/))

![image](https://cloud.githubusercontent.com/assets/1690934/8641002/400d58ba-2903-11e5-91dd-e27947d71883.png)

The grid layout looks good when posters are present, I'm guessing it'll perform poorly without any graphics.

**Comment by BenWoodford76** [comment]
I quite like the grid layout idea, problem would be for older shows where we don't have a poster
